.travis/cairo-windows.py

```diff
@@ -0,0 +1,51 @@
+import platform
```

Comment:
Is this the same script as in #71? If so, please make sure to use that script once #71 is merged, because when it does, this script will be in two different places.